src/sidemenu/PinnedMenu.tsx

```diff
@@ -117,11 +137,11 @@ const Section: React.FC<PinnedMenuSectionProps> = (props) => {
   } = props as LoadingPinnedSectionProps;
 
   return (
-    <div className="pinned-section">
-      <div className="icon">
+    <section>
+      <div className="icon"  role="img" aria-label={loading? `loading ${sectionId}` : sectionId}>
```

Comment:
Oh, this is interesting, cause I would assume `role="img"` should go into `Icon`.

I would suggest making `Icon` itself more accessible by adding the label there. https://github.com/essential-randomness/bobaboard-ui/blob/main/src/common/Icon.tsx

Doesn't have to be in this review, but can you add a TODO?

Reply:
The actual svgs from font awesome already have `role="img"`, so I don't think `Icon` actually needs it as well, but Storybook's accessibility checker got mad about having an `aria-label` on a `div` without a `roll`, so that's why I added it here. I can remove it if that would be better?

Reply:
If you add a "label" prop to the Icon component then you should be able to just pass the label to the font awesome icon. I think we should figure out how to go this route rather than having a role img within another role img.

Reply:
Okay, figured it out. After some digging it looks like Font Awesome calls their prop for adding a label `title`